curriculum/challenges/english/15-javascript-algorithms-and-data-structures-22/learn-basic-javascript-by-building-a-role-playing-game/62a3cdb11478a34ff4a6470d.md

```diff
@@ -0,0 +1,152 @@
+---
+id: 62a3cdb11478a34ff4a6470d
+title: Step 38
+challengeType: 0
+dashedName: step-38
+---
+
+# --description--
+
+Now create a `fightDragon` function that prints `Fighting dragon.` to the console.
+
+Once you have done that, open your console and try clicking the buttons on your project.
```

Comment:
In the console I am getting this error, because the script is loaded before our button elements because we placed the script element in the head. 
<img width="763" alt="Screen Shot 2022-07-10 at 8 17 29 PM" src="https://user-images.githubusercontent.com/67210629/178182257-09891f30-083e-4489-bd73-4e48d27fb703.png">